certbot/updater.py

```diff
@@ -29,8 +29,8 @@ def run_generic_updaters(config, lineage, plugins):
         return
     try:
         # installers are used in auth mode to determine domain names
```

Comment:
I think this comment is outdated as I believe it was referring to the use of "certonly".

Reply:
That's right, removed.